go.mod

```diff
@@ -3,22 +3,17 @@ module github.com/haproxytech/client-native
 go 1.12
 
 require (
-	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a // indirect
-	github.com/go-openapi/analysis v0.19.0 // indirect
-	github.com/go-openapi/errors v0.19.0
-	github.com/go-openapi/jsonpointer v0.19.0 // indirect
-	github.com/go-openapi/jsonreference v0.19.0 // indirect
-	github.com/go-openapi/loads v0.19.0 // indirect
-	github.com/go-openapi/runtime v0.19.0 // indirect
-	github.com/go-openapi/spec v0.19.0 // indirect
+	github.com/go-openapi/analysis v0.19.3 // indirect
+	github.com/go-openapi/errors v0.19.2
+	github.com/go-openapi/runtime v0.19.2 // indirect
 	github.com/go-openapi/strfmt v0.19.0
+	github.com/go-openapi/swag v0.19.3 // indirect
 	github.com/google/uuid v1.1.1
-	github.com/haproxytech/config-parser v0.9.1
-	github.com/haproxytech/models v1.0.1-0.20190701143542-2dd789d222df
-	github.com/mailru/easyjson v0.0.0-20190403194419-1ea4449da983 // indirect
+	github.com/haproxytech/config-parser v1.0.0
+	github.com/haproxytech/models v1.0.1-0.20190701144201-4e7b9b418f32
+	github.com/mailru/easyjson v0.0.0-20190626092158-b2ccc519800e // indirect
```

Comment:
Here you should only update the versions of the packages you need, in your case config-parser and models, you can do that by just doing:

GO111MODULE=on go get github.com/haproxytech/models@<your_commit>

Reply:
done